src/config/mgdLDefaults.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    insulinSensitivity: {
+        defaultStart: 500
```

Comment:
[nit] If this is in the defaults, then using a default prefix here is redundant.